package.json

```diff
@@ -30,22 +29,22 @@
   "homepage": "https://github.com/snyk/snyk-go-parser#readme",
   "dependencies": {
     "@iarna/toml": "^2.2.5",
-    "@snyk/dep-graph": "^2.3.0",
+    "@snyk/dep-graph": "^2.6.1",
+    "@snyk/go-semver": "^1.2.1",
```

Comment:
do we want to be more specific here?

Reply:
as in pin?

Reply:
maybe, CLI could relock in a break potentially